lib/eksekuter.rb

```diff
@@ -6,9 +6,11 @@
 
 # Class that command execution is delegated to by Eksek.
 class Eksekuter
-  def initialize cmd, opts
-    @cmd = cmd
-    @opts = opts.freeze
+  def initialize *args, **opts
+    @env = args[0].is_a?(Hash) ? args.shift : {}
+    @env = @env.each_with_object({}) { |(k, v), o| o[k.to_s] = v }
```

Comment:
💖

Reply:
There's also `@env.transform_keys(&:to_s)` which is equivalent to `@env.transform_keys { |k| k.to_s }`, but it's not available in earlier version of Ruby.